samples/constitutions/trusted_authority/resolve.js

```diff
@@ -0,0 +1,81 @@
+function getMemberInfo(memberId) {
+  return ccf.bufToJsonCompatible(
+    ccf.kv["public:ccf.gov.members.info"].get(ccf.strToBuf(memberId))
+  );
+}
+
+function isRecoveryMember(memberId) {
+  return (
+    ccf.kv["public:ccf.gov.members.encryption_public_keys"].get(
+      ccf.strToBuf(memberId)
+    ) ?? false
+  );
+}
+
+// Defines which of the members are operators.
+function isOperator(memberId) {
+  // Operators cannot be recovery members.
+  if (isRecoveryMember(memberId)) {
+    return false;
+  }
+  return getMemberInfo(memberId).member_data?.is_operator ?? false;
+}
+
+// Defines which of the members are trusted authorities.
+function isTrustedAuthority(memberId) {
+  return getMemberInfo(memberId).member_data?.is_trusted_authority ?? false;
```

Comment:
Operator provisioning authorities also should not be recovery members (see line 18).